### PR TITLE
Mes 6994 todo clean up

### PR DESCRIPTION
**Pull request by gittins7**
## Description
Removed no longer applicable TODO's, and amended any TODO's that needed updating
## Checklist

- [ ] PR title includes the JIRA ticket number
- [ ] Branch is rebased against the latest develop
- [ ] Code has been tested manually
- [ ] PR link added to JIRA ticket
- [ ] One review from each scrum team
- [ ] Squashed commit contains the JIRA ticket number

### CODE REVIEW

File: src/app/pages/test-report/test-report.analytics.effects.ts [hunk 1]
```diff
@@ -1128,7 +1128,7 @@ export class TestReportAnalyticsEffects {
     }),
   ));
 
-  // @TODO - MES-6879 - enable once component migrated
+  // @TODO - MES-6879 - enable once component migrated | Cat B Dependency
```
Comment (andrewsetterfield) [inline src/app/pages/test-report/test-report.analytics.effects.ts:1131]: This story is marked as done. Did we say Cat X+E uses this and it needs a new story

File: src/app/pages/test-report/test-report.analytics.effects.ts [hunk 2]
```diff
@@ -1147,7 +1147,7 @@ export class TestReportAnalyticsEffects {
   //   }),
   // );
 
-  // @TODO - MES-6879 - enable once component migrated
+  // @TODO - MES-6879 - enable once component migrated | Cat B Dependency
```
Comment (andrewsetterfield) [inline src/app/pages/test-report/test-report.analytics.effects.ts:1150]: This story is marked as done. Did we say Cat X+E uses this and it needs a new story

**Review by andrewsetterfield — CHANGES_REQUESTED** [review]
Didnt we discuss creating new stories with Luke - i expected to see those story numbers added to the existing TODO's so we could keep track of them

File: src/app/pages/test-report/test-report.analytics.effects.ts (hunk 1 above)
Comment (gittins7) [reply src/app/pages/test-report/test-report.analytics.effects.ts:1131]: Yes i believe that is correct

File: src/app/pages/test-report/test-report.analytics.effects.ts (hunk 2 above)
Comment (gittins7) [reply src/app/pages/test-report/test-report.analytics.effects.ts:1150]: Same as previous comment

**Pull request closed without merging**